src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 206px 16px 16px;
+  box-sizing: border-box;
```

Comment:
move to `*` selector

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3425/files#r1293460457